src/System.IO.FileSystem/src/System/IO/Win32FileSystem.cs

```diff
@@ -46,11 +46,7 @@ public override void CopyFile(string sourceFullPath, string destFullPath, bool o
 
         public override void ReplaceFile(string sourceFullPath, string destFullPath, string destBackupFullPath, bool ignoreMetadataErrors)
         {
-            int flags = Interop.Kernel32.REPLACEFILE_WRITE_THROUGH;
-            if (ignoreMetadataErrors)
-            {
-                flags |= Interop.Kernel32.REPLACEFILE_IGNORE_MERGE_ERRORS;
-            }
+            int flags = ignoreMetadataErrors ? Interop.Kernel32.REPLACEFILE_IGNORE_MERGE_ERRORS : 0;
```

Comment:
Ah, MSDN says it's not supported - I assume this means it is ignored. If so, you can remove where we define REPLACEFILE_WRITE_THROUGH above